api/src/tests/transactions_test.rs

```diff
@@ -874,9 +890,14 @@ async fn test_get_txn_execute_failed_by_invalid_script_function_name() {
         context,
         account,
         "0x1",
-        "TestCoin",
+        "Coin",
         "transfer_invalid",
-        vec![],
+        vec![TypeTag::Struct(StructTag {
+            address: AccountAddress::ONE,
+            module: ident_str!("TestCoin").to_owned(),
+            name: ident_str!("TestCoin").to_owned(),
+            type_params: vec![],
```

Comment:
it might be good to make this into something in types along the lines of `types/src/utility_coin.rs`

Reply:
Updated.